atd-vzv/src/views/dashboard/helpers/helpers.js

```diff
@@ -0,0 +1,36 @@
+// Helpers to handle Socrata responses for Summary view components
+
+export const calculateTotalFatalities = data => {
+  let total = 0;
+  data.forEach(record => (total += parseInt(record.death_cnt)));
+  return total;
+};
+
+export const calculateTotalInjuries = data => {
+  let total = 0;
+  data.forEach(record => (total += parseInt(record.sus_serious_injry_cnt)));
+  return total;
+};
+
+export const calculateTotalCrashes = data => {
+  let total = 0;
+  data.forEach(record => (total += 1));
+  return total;
+};
+
+export const getYearsOfLifeLost = fatalityData => {
+  // Assume 75 year life expectancy,
+  // Find the difference between person.prsn_age & 75
+  // Sum over the list of ppl with .reduce
+  return fatalityData.reduce((accumulator, fatalityRecord) => {
+    let years = 0;
+    if (fatalityRecord.prsn_age !== undefined) {
+      let yearsLifeLost = 75 - Number(fatalityRecord.prsn_age);
```

Comment:
Is the number 75? For some reason I thought it was different but I'm probably wrong. Could we maybe save this value as a constant and import in case it ever changes?

Reply:
Yep, this was taken directly from VZE. I'll make a new constants file for this type of data in PR PR #603 so this won't remain buried in here.